web/templates/settings/edit_bot_form.hbs

```diff
@@ -1,4 +1,4 @@
-<div id="bot-edit-form" data-user-id="{{user_id}}" data-email="{{email}}">
+<div id="bot-edit-form" data-user-id="{{user_id}}" data-email="{{email}}" data-simplebar>
```

Comment:
Do we need this? We already have `data-simplebar` attribute set to `modal__content` element.